pages/knowledge/colours-and-contrast/how-to-calculate/README.md

```diff
@@ -0,0 +1,57 @@
+---
+navigation_title: "How to calculate"
+position: 1
+changed: "2019-04-21"
+---
+
+# How to calculate colour contrast
+
+**There is a mathematical formula to calculate the contrast ratio between two colours. Using this formula for contrast evaluation is always better than relying only on your eyes.**
+
+Please note that we are keeping technical explanations simple here. If you want to read a detailed definition of the colour contrast formula, please go to [contrast ratio (W3.org)](https://www.w3.org/TR/WCAG20/#contrast-ratiodef).
+
+## Relative luminance
+
+Contrast is calculated using “relative luminance”, which is defined as:
+
+> The relative brightness of any point in a colorspace, normalized to `0` for darkest black and `1` for lightest white.
+
+Quote: [relative luminance (W3.org)](https://www.w3.org/TR/WCAG20/#relativeluminancedef)
+
+![Relative luminance gradient](_media/relative-luminance-gradient.png)
+
+The relative luminance can be calculated from any colour code (like HEX or RGB).
+
+## The formula
+
+To calculate the contrast ratio, the relative luminance of the lighter colour (`L1`) is divided through the relative luminance of the darker colour (`L2`), like so:
+
+    (L1 + 0.05) / (L2 + 0.05)
+
+This results in a value ranging from `1:1` (no contrast at all) to `21:1` (the highest possible contrast). While high contrast is generally good, higher is not always better. Research indicates that a moderate contrast somewhere between the extremes may be best.
+
```

Comment:
I'd love to see this sentence here again (instead of the link at the bottom of the page):

> Fortunately, we do not need to calculate anything of this manually ourselves when using an appropriate tool. Read on for more details.

Reply:
**To Do:** Mention that you don’t have to calculate yourself.

Reply:
Don’t

Reply:
Aaah stupid… *Done* is what I meant to say